### PR TITLE
adding typescript script for ubi 8.5

### PR DESCRIPTION
**Pull request by adilhusain-s**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: t/typescript/typescript_3.9.9_ubi_8.3.sh [hunk 1]
```diff
@@ -0,0 +1,62 @@
+#-----------------------------------------------------------------------------
+#
+# Package	:typescript
+# Version	: 3.9.9
+# Source repo	: https://github.com/Microsoft/TypeScript
+# Tested on	: ubi 8.3
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Adilhusain Shaikh <Adilhusain.Shaikh@ibm.com>
+#
```
Comment (seth-priya) [inline t/typescript/typescript_3.9.9_ubi_8.3.sh:9]: @adilhusain-s We have recently added some additional fields here like Language and Travis-Check, please update/add those, refer to the latest templates at https://github.com/ppc64le/build-scripts/tree/master/templates

File: t/typescript/typescript_3.9.9_ubi_8.3.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME="TypeScript"
+PACKAGE_VERSION="v3.9.9"
```
Comment (Siddhesh-Ghadi) [inline t/typescript/typescript_3.9.9_ubi_8.3.sh:21]: Make this configurable from command line.
```suggestion
PACKAGE_VERSION=${1:-v3.9.9}
```

**Review by Siddhesh-Ghadi — CHANGES_REQUESTED** [review]

**Comment by Siddhesh-Ghadi** [comment]
Tests are failing on travis with timeout. Are they passing on local setup? @adilhusain-s 
```
 fourslash test annotateWithTypeFromJSDoc1.ts runs correctly:\n     Error: Timeout of 40000ms exceeded.
```

**Comment by adilhusain-s** [comment]
yes, the build and tests are successful on local set up inside the container.

**Pull request merged**